sponsors/models/sponsorship.py

```diff
@@ -117,6 +117,18 @@ def clone(self, year: int):
             slug=self.slug, year=year, defaults=defaults
         )
 
+    def get_default_revenue_split(self) -> list[tuple[str, float]]:
+        """
+        Give the admin an indication of how revenue for sponsorships in this package will be divvied up
+        """
+        values, key = {}, "program__description"
```

Comment:
```suggestion
        values, key = {}, "program__name"
```

This is a required field, so I recommend using that.. Description is not provided for most of the packages!